tools/generator/dfg/stm32/stm.py

```diff
@@ -89,6 +89,71 @@ def getGpioRemapForModuleConfig(module, config):
         mmm['mapping'] = stm32f1_gpio_remap[module]['mapping'][int(config)]
     return mmm
 
+stm32f3_dma_remap = \
+{
+    'dma1ch1': {
+        'tim17_ch1': 'tim17_up',
+        'tim17_up':  {'position': 12, 'clear': 1, 'set': 0},
+    },
+    'dma1ch2': {
+        'adc2':      {'position': 72, 'clear': 3, 'set': 2},
+        'i2c_tx':    {'position': 70, 'clear': 3, 'set': 1},
+        'spi_rx':    {'position': 64, 'clear': 3, 'set': 0}, # also 'set': 3
```

Comment:
I don't see an advantage for also adding the 0b11 encoding here, it does the same thing as the 0b00 encoding.